tools/data_integrity/score_instrument.php

```diff
@@ -0,0 +1,225 @@
+<?php
+/**
+ * this tools scores any registered instrument that was build using the
+ * NDB_BVL_Instrument class and that has a working score() method.
+ * the cmd-line arguments need to contain a valid test_name and 'all' or 'one' and a
```

Comment:
**The command line** arguments need to contain a valid **test_name, 'all' or 'one' with a CandID and SessionID.** 'all' scores all records of an instrument that belong to an active **timepoint with** data entry **that has been** completed and administration **not set to** None.